qiskit_aer/backends/aerbackend.py

```diff
@@ -351,17 +342,16 @@ def max_circuits(self):
 
     @property
     def target(self):
-        self._target = convert_to_target(
-            self.configuration(), self.properties(), self.defaults(), self._mapping
-        )
-        return self._target
+        if self._target is not None:
+            return self._target
+
+        return convert_to_target(self.configuration(), self.properties(), None, NAME_MAPPING)
```

Comment:
This is just a minor comment. Previously, converted `target` was cached. With this change, it will not be cached. I think frequent access of `target` property will produce performance regression. Introducing caching may help performance (though setting `method` may need clear cache of `target`).

Reply:
because if target is cached and used for transpilation to multiple circuits, basis gates for the first circuit is reused that can be a problem if the second circuit has gate that the first one does not have